packages/frontend/web/components/elements/RichLinkComponent.tsx

```diff
@@ -45,10 +46,20 @@ const richLinkPillarColour: (pillar: Pillar) => colour = pillar => {
 };
 
 const richLinkContainer = css`
-    width: 8.125rem;
+    ${until.wide} {
```

Comment:
hmm it's weird that these changes are showing up here when they were made on my branch/merge

Reply:
That's me merging in master to fix conflicts. Sorry, I should have waiting for the reviews